setup.py

```diff
@@ -1,25 +1,25 @@
 from setuptools import setup, find_packages
+from pkg_resources import parse_requirements
+from pathlib import Path
 
-with open("./README.md") as fhandle:
-    readme = fhandle.read()
+with Path('requirements.txt').open() as requirements_txt:
+    install_requires = [
+        str(requirement)
+        for requirement
+        in parse_requirements(requirements_txt)
+    ]
 
+# TODO: fix tests incorrectly running when installing with setup.py
 setup(
     name="make_prg",
-    version="0.2.0",
-    description="Build genome graphs (PRGs) from multiple sequence alignments",
-    long_description=readme,
-    long_description_content_type="text/markdown",
+    version="0.4.0",
     packages=find_packages(),
-    url="https://github.com/iqbal-lab-org/make_prg",
+    url="https://github.com/rmcolq/make_prg",
```

Comment:
No, we transferred this repo to the lab url https://github.com/iqbal-lab-org/make_prg